src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,62 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_LENGTH = 6;
+    private static final int MINIMAL_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws RuntimeException {
+        if (user == null) {
+            throw new InvalidUserException("User can't be null!");
+        }
```

Comment:
add checks `user.getAge() == null` `user.getPassword() == null`

Reply:
There are already boolean checks below. Or what do you mean?